### PR TITLE
Quick connect wallet

### PR DESCRIPTION
**Pull request by razacodespython**

### CODE REVIEW

File: portal/guides/connect-wallet.mdx [hunk 1]
```diff
@@ -21,34 +21,33 @@ If you don't have a a wallet, click [here](/guides/create-a-metamask-wallet). If
 
 ```bash
 //Run this to install with npm
-npm install @3rdweb/react @3rdweb/hooks
+npm install @thirdweb-dev/react @thirdweb-dev/hooks
```
Comment (nachoiacovino) [inline portal/guides/connect-wallet.mdx]:
```suggestion
npm install @thirdweb-dev/react
```

hooks package doesn't exist

File: portal/guides/connect-wallet.mdx [hunk 2]
```diff
@@ -21,34 +21,33 @@ If you don't have a a wallet, click [here](/guides/create-a-metamask-wallet). If
...
 
+Create a new folder called components that will host all custom components of your app.
 Inside your `components` folder, create a file `Connect.js` with the following code 👇
 
-By wrapping it in the `<nav>` tag, the button will appear in the nav bar.
 
-```jsx:Connect.js
-import { ConnectWallet } from "@3rdweb/react";
+```js:Connect.js
+import { useDisconnect, useMetamask } from "@thirdweb-dev/react";
```
Comment (nachoiacovino) [inline portal/guides/connect-wallet.mdx]:
```suggestion
import { useMetamask } from "@thirdweb-dev/react";
```

File: portal/guides/connect-wallet.mdx [hunk 3]
```diff
@@ -92,10 +68,7 @@ So make sure you put your `Component` within the brackets of the `ThirdwebProvid
 
 ```jsx:App.js
 	return (
-    <ThirdwebProvider
-      connectors={connectors}
-      supportedChainIds={supportedChainIds}
-    >
+    <ThirdwebProvider desiredChainId={supportedChainIds}>
       <Connect /> /* our connect wallet button */
```
Comment (nachoiacovino) [inline portal/guides/connect-wallet.mdx]:
```suggestion
      <Connect /> /* our connect wallet button */
```
that's a syntax error, not a valid way of commenting in react

**Review by nachoiacovino — CHANGES_REQUESTED** [review]
when you link to the connect to blockchain guide the link is broken

this one is mostly fine.

File: portal/guides/connect-wallet.mdx (hunk 3 above)
Comment (razacodespython) [reply portal/guides/connect-wallet.mdx]: adjusted

**Comment by razacodespython** [comment]
@nachoiacovino lmk if this can be merged

**Comment by nachoiacovino** [comment]
Killing this guide in favor of the claim-button-react one.

**Pull request closed without merging**